programs/programs/urgent_needs/tx/early_intervention.py

```
@@ -0,0 +1,10 @@
+from ..base import UrgentNeedFunction
+
+
+class EarlyIntervention(UrgentNeedFunction):
+    dependencies = ["age", "county"]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)